bin/xdebug-enable

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+sudo rm -f /usr/local/etc/php/conf.d/*blackfire.ini
+sudo bash -c 'echo "zend_extension=xdebug.so" > /usr/local/etc/php/conf.d/xdebug.ini'
```

Comment:
```suggestion
sudo sed -i 's/^#zend_extension=\(.*\)/zend_extension=\1/' /usr/local/etc/php/conf.d/xdebug.ini
```

Reply:
If `PHP_DEBUGGER=none` is set the file is none existent, so we delete and create it as well.